src/hooks.js

```diff
@@ -50,16 +50,10 @@ const hooks = [
         const parsed = path.parse(file);
         // Only write the file/folder structure if it has an extension
         if (parsed.ext && parsed.ext.length > 0) {
-          const relativeToAssetsArray = parsed.dir.split('assets');
-          relativeToAssetsArray.shift();
-
-          const relativeToAssetsFolder = `.${relativeToAssetsArray.join()}/`;
-          const p = path.parse(path.resolve(settings.distDir, relativeToAssetsFolder));
-          fs.ensureDirSync(p.dir);
-          fs.outputFileSync(
-            path.resolve(settings.distDir, `${relativeToAssetsFolder}${parsed.base}`),
-            fs.readFileSync(file),
-          );
+          const relativeToAssetsFolder = path.relative(path.join(settings.rootDir, './src/assets'), file)
```

Comment:
use `path.relative` to do what youre trying to do here

Reply:
TIL. Thank you. 👍